columnphysics/icepack_atmo.F90

```diff
@@ -359,8 +359,16 @@ subroutine atmo_boundary_layer (sfctype,            &
       ! as in Jordan et al (JGR, 1999)
       !------------------------------------------------------------
 
-      shcoef = rhoa * ustar * cp * rh + c1
-      lhcoef = rhoa * ustar * Lheat  * re
+      if (trim(atmbndy) == 'mixed') then
+         !- Use constant coefficients for sensible and latent heat fluxes
+         !    similar to atmo_boundary_const but using vmag instead of wind
+         shcoef = (1.20e-3_dbl_kind)*cp_air*rhoa*vmag
+         lhcoef = (1.50e-3_dbl_kind)*Lheat *rhoa*vmag
```

Comment:
Please remove literal constants from the code (here 1.20e-3_dbl_kind  and 1.50e-3_dbl_kind) and place them in the Icepack constants file.  Also, here, how does this work if `sfctype(1:3)=='ocn'`?

Reply:
Introduce p0012, p0015, which makes sense.